### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by Eugeniusz-Gienek**
I have changed the name of my repository from WindyWanderer to Eugeniusz-Gienek - there is already Github redirect in place, but the correct behaviour would be to update all of the links which use it.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @Eugeniusz-Gienek. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]

**Pull request merged**
